src/lib/components/avatar.svelte

```diff
@@ -20,11 +17,11 @@
 	tabindex={0}
 >
 	<div class="img" style={`height: ${size}px;`}>
-		{#if picture && isCID(picture)}
+		{#if picture}
 			<img src={getPicture(picture)} alt="profile" />
-		{:else if picture}
+		{:else if seed}
 			{@const style = group ? 'shapes' : 'bottts-neutral'}
-			<Dicebear name={picture || ''} {size} {style} />
+			<Dicebear {seed} {size} {style} />
 		{/if}
```

Comment:
Can it happen that both `picture` and `seed` is undefined? I guess then we only see grey circle, right? I can live with that.

Reply:
Normally this cannot happen, because there is always an address to be used for seed. But yes if this is the case then the result is a gray circle.